commerce/src/wcs.js

```diff
@@ -32,12 +33,12 @@ const WcsBaseUrl = {
  */
 export function Wcs({ settings }) {
     const log = Log.module('wcs');
-    const { env, wcsApiKey: apiKey } = settings;
-
+    const { env, domainSwitch, wcsApiKey: apiKey } = settings;
+    const baseUrl = domainSwitch ? WcsBaseUrl['STAGE_ACOM'] : WcsBaseUrl[env];
```

Comment:
this is problematic in case you are domainSwitch on and env production, no? i guess we won't have both normally, but still looks weird

Reply:
i added acom prod too